File: Python Course/Software Development Module - in Python/Revision/Players/FullBackPlayer.py
"""
FullBackPlayer.py
Billy Cussen
26/10/2020
"""
import FootballPlayer

class myFullBackPlayer(FootballPlayer.myFootballPlayer):

    def __init__(self, name):
        super().__init__(name)

    def getRole(self):
        return "Hi "+str(self.name)+", I am a Fullback Player"

File: Java/Software Development Module - Java/Revision/Players/FullBackPlayer.java
/*
FullBackPlayer.java
Billy Cussen
28/07/2020
*/

public class FullBackPlayer extends DefensivePlayer{

    //Constructor
    public FullBackPlayer(String name){
        super(name);
    }

    @Override
    public String role(){
        return "Hi, I'm "+name+". I am a Full Back.";
    }

}